### PR TITLE
Measure inactiveness

### LINKED ISSUE #429: Better measurement for "inactive" resources
In order to better understand which repositories are "inactive" (right now: every repo where all users with access to it did not sign into Zappr for a year), I would propose the metric of "time since last check execution without errors". In order to change our clean up job, we would need two columns `last_executed` and `last_execution_successful` that are set accordingly.

### PR DESCRIPTION
**Pull request by prayerslayer**
Fixes #429 

Added four columns to `checks` table:

~~~
select last_execution_delay_ms as delay, last_execution_ms as exectime, last_execution_ts as exec, last_execution_successful as success from zappr_data.checks;
 delay | exectime |             exec              | success 
-------+----------+-------------------------------+---------
  1893 |      880 | 2016-12-07 16:53:20.93357+00  | t
(1 row)
~~~

Tests I did:

* Start locally with localtunnel and postgres, trigger `pullrequestlabels` check, do above query.

### CODE REVIEW

File: common/util.js [hunk 1]
```diff
@@ -96,6 +96,27 @@ export function setIn(obj, path, value) {
...
+
+/**
+ * Returns an array of [key, val] for all
+ * keys in provided object where fn(key) returns true.
+ *
+ * @param obj object to walk
+ * @param fn will be passed key. is expected to return true iff key/ should be talken.
+ * @param result don't use
+ */
+export function findDeepInObj(obj, fn, result = []) {
```
Comment (sebastianpoeplau) [inline common/util.js]: Is it accepted in JS to have parameters like `result` that are not to be used by callers? (You can see my total ignorance of all thing JavaScript...)

File: test/server/checkrunner.test.js [hunk 2]
```diff
@@ -135,8 +145,33 @@ describe('CheckRunner', () => {
...
+        done()
+      } catch (e) {
+        done(e)
+      }
+    })
+
     CHECK_TYPES.forEach(type => {
-      it(`[${type}] merges token into provided args and calls execute`, async(done)=> {
+      it(`[${type}] merges token into provided args and calls execute`, async(done) => {
+        console.log(type)
```
Comment (sebastianpoeplau) [inline test/server/checkrunner.test.js]: Is this a leftover from development or do you actually want to print the types?

**Comment by sebastianpoeplau** [comment]
👍

**Comment by sebastianpoeplau** [comment]
👍

**Comment by prayerslayer** [comment]
👍

**Pull request merged**